en/docs/how-tos/analytics-for-microgateway.md

```diff
@@ -60,7 +60,13 @@ Real-Time data publishing has the following advantages:
 
 ### Configuring Analytics for the Microgateway
 
-The following sections describe how to configure the WSO2 API-M Analytics Server for Microgateway.
+The following sections describe how to configure the WSO2 API-M Analytics Server for Microgateway. When adding the relevant analytics configurations to `<MICROGW_HOME>/conf/micro-gw.conf` file, make sure to add the version of the API-M Analytics you are using from the following config.
+
+```toml
+[analytics]
+  # The configured API Manager analytics stream version
+  streamVersion = "3.2.0"
+```
 
 #### Using periodical data publishing (file-based)
```

Comment:
Lets metion here this config is only required if you are choosing the file based analytics publishing